google-analytics-chart.html

```diff
@@ -38,13 +56,17 @@
     </google-chart>
     <content select="footer"></content>
   </template>
+</dom-module>
 
-  <script>
+<script>
 
-    (function() {
+  (function() {
 
-      Polymer('google-analytics-chart', {
+    Polymer( {
 
+      is: 'google-analytics-chart',
+
+      properties: {
```

Comment:
There is inconsistent newline spacing for the properties below. The style guide doesn't address spacing between properties, but I think they should at least be consistent.